ams/dashboard/src/components/tables-sub-menu/TablesMenu.vue

```diff
@@ -247,6 +252,13 @@ export default defineComponent({
       if (!state.curCatalog) {
         return
       }
+      if (state.allDatabaseListLoaded.length) {
+        state.databaseList = state.allDatabaseListLoaded.filter((ele) => {
+          return ele.label.includes(state.DBSearchInput)
+        })
+        return
+      }
+
```

Comment:
Maybe can use `computed` to implement this filtering instead of creating a new state

Reply:
But we have to store all databases/tables in the frontend, then filtering by keyword can be acted at them. the databaseList and tableLIst in the state just are just used as values displaying in `virtual-recycle-scroller`

Reply:
There is no conflict between the two. You just need to remove database and table and filter with computed.

Reply:
@zhangmo8 Can you review it again?